devices/salus_controls.js

```diff
@@ -83,7 +83,21 @@ module.exports = [
         description: 'Door or window contact sensor',
         fromZigbee: [fz.ias_contact_alarm_1],
         toZigbee: [],
-        exposes: [e.contact(), e.battery_low(), e.tamper()],
+        exposes: [e.contact()],
```

Comment:
Why was `e.battery_low(), e.tamper()` removed here? `fz.ias_contact_alarm_1` supports these.

Reply:
That was a fat fingered fumble. I was changing the PS600 line and somehow modified the OS600 line too. Sorry about that, I'll resubmit.